gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/KafkaAvroJobStatusMonitor.java

```diff
@@ -163,6 +163,9 @@ public GobblinTrackingEvent deserializeEvent(DecodeableKafkaRecord<byte[],byte[]
         properties.put(JobStatusRetriever.EVENT_NAME_FIELD, ExecutionStatus.CANCELLED.name());
         properties.put(TimingEvent.JOB_END_TIME, properties.getProperty(TimingEvent.METADATA_END_TIME));
         break;
+      case TimingEvent.JOB_COMPLETION_PERCENTAGE:
+        properties.put(TimingEvent.JOB_LAST_PROGRESS_PERCENT_TIME, properties.getProperty(TimingEvent.METADATA_END_TIME));
```

Comment:
Is METADATA_END_TIME the correct value here?

Reply:
I believe METADATA_END_TIME is set when event.stop() is called (which is what I use to obtain the timestamp of when the progress event was set)